app/assets/stylesheets/views/article.scss

```diff
@@ -0,0 +1,364 @@
+    font-size: $fs-s;
+    display: flex;
+  }
+
+  &__body {
+    font-size: $fs-l;
+    font-family: $ff-serif;
+    @media (min-width: $breakpoint-m) {
+      font-size: $fs-xl;
+    }
+
+    a {
+      code {
+        color: var(--link-color);
+      }
+
+      &.anchor {
+        padding-top: 0;
+        margin-top: 0;
+        top: -50px;
+        display: block;
+        visibility: hidden;
+        position: relative;
+      }
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin: var(--article-rythm) 0;
+      margin-top: calc(var(--article-rythm) * 2);
+      padding: 0;
+    }
+
+    h1 {
+      font-size: 1.85em;
+      line-height: 1.25em;
+      font-weight: 400;
+    }
+
+    h2 {
+      font-size: 1.7em;
+      font-weight: 400;
+      line-height: 1.25em;
+      margin-bottom: 0.5em;
+    }
+
+    h3 {
+      font-size: 1.32em;
+      font-weight: 400;
+    }
+
+    hr {
+      width: 25%;
+      opacity: 0.1;
+      border: 1px solid var(--body-color);
+      margin: 1.3em auto 1.5em;
+    }
+
+    p {
+      margin: var(--article-rythm) 0;
+    }
+
+    ul,
+    ol {
+      padding-left: $su-6;
+      margin: var(--article-rythm) 0;
+      br {
+        line-height: 0;
+        margin: 0;
+        padding: 0;
+      }
+    }
+
+    ul {
+      list-style-type: disc;
+
+      p {
+        margin: 0;
+        padding: 0;
+      }
+
+      ul {
+        margin: 0 0.7em 0 1.9em;
+        list-style-type: square;
+
+        ul {
+          list-style-type: circle;
+        }
+      }
+    }
+
+    li {
+      margin: $su-2 0;
+    }
+
+    figure {
+      margin: 0;
+    }
+
+    figcaption {
+      font-style: italic;
+      text-align: center;
+      font-size: 0.8em;
+      line-height: 1.4em;
+      color: var(--body-color);
+      opacity: 0.75;
+      display: block;
+    }
+
+    p + figcaption {
+      margin-top: -0.8em;
+    }
+
+    blockquote {
+      border-left: 4px solid var(--base-50);
+      padding: 0 $su-4;
+      margin: var(--article-rythm) 0;
+
+      p:first-child {
+        margin: $su-2 0;
+      }
+
+      &.twitter-tweet {
+        font-family: $ff-default;
+        background: var(--body-color-inverted);
+        max-width: 456px;
+        border: 1px solid #e9eef2;
+        font-size: 0.75em;
+        line-height: 1.35em;
+        border-radius: 5px;
+        margin: 1.6em auto;
+        padding: 60px 10px 18px;
+        color: white;
+        min-height: 60px;
+        padding: 33px 15px 8px;
+
+        a {
+          color: white;
+        }
+
+        @media screen and (min-width: $breakpoint-s) {
+          padding: 60px 22px 18px;
+          min-height: 105px;
+        }
+      }
+    }
+
+    code {
+      margin: auto;
+      background: rgba(0, 0, 0, 0.1);
+      padding: 0.1em 0.3em 0;
+      border-radius: 2px;
+      font-size: 0.84em;
+      vertical-align: 0;
+      max-width: 100%;
+      line-height: 1.6em;
+      word-break: break-all;
+    }
+
+    kbd {
```

Comment:
what is kbd?

Reply:
html element for representing keys on keyboard. literally :) https://www.w3schools.com/tags/tag_kbd.asp